src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -132,6 +132,9 @@ class oopDesc;
 #define UINT64_FORMAT_X_0        "0x%016"     PRIx64
 #define UINT64_FORMAT_W(width)   "%"   #width PRIu64
 
+// Format without leading 0x
+#define INT64_FORMAT_X_0_        "%016"       PRIx64
```

Comment:
Why the trailing underscore?

The name doesn't make sense to me given we have UINT64_FORMAT_X_0 and the only difference here is the leading `0x` which doesn't really have anything to do with signed-ness. Maybe `RAW_UINT64_FORMAT_X_0` to indicate the dropped `0x` prefix ?

Reply:
Thanks for the suggestion.  All the macros start with UINT64_FORMAT though.  How about UINT64_FORMAT_RAW_X_0 ?  or UINT64_FORMAT_X_0_RAW ?  Still not great.

Reply:
UINT64_FORMAT_RAW_X_0  is not terrible. :)

Reply:
To be honest though I don't know why we want hexadecimal output without the leading 0x ...

Reply:
The `_X` infix is meant to indicate `0x`. I think `UINT64_FORMAT_0` is what you are looking for.

Reply:
I thought X was hexadecimal.  That's a lot better.  Thanks.

Reply:
```
// Guide to the suffixes used in the format specifiers for integers:
//        - print the decimal value:                   745565
//  _X    - print as hexadecimal, without leading 0s: 0x12345
//  _X_0  - print as hexadecimal, with leading 0s: 0x00012345
```
It wasn't obvious to me that the _X meant hexadecimal-with-0x-prefix versus simply hexadecimal. But I have no issue with always doing 0x :)

Reply:
Hmm. You are right. I read too much into the comment (that I wrote some time ago :) )